File: Programmers/Lv.0/배열 만들기 2/181921.java
import java.util.List;
import java.util.ArrayList;

class Solution {
    public int[] solution(int l, int r) {
        int[] answer = {};
        
        List<Integer> list = new ArrayList<>();
        for (int i=l; i<=r; i++) {
            String s = String.valueOf(i);
            if ("".equals(s.replaceAll("5", "").replaceAll("0", ""))) {
                list.add(i);
            }
        }
        
        if (list.size() == 0) {
            answer = new int[] {-1};
        } else {
            answer = new int[list.size()];
            for (int j=0; j<list.size(); j++) {
                answer[j] = list.get(j);
            }
        }
        
        return answer;
    }
}

File: Programmers/Lv.0/배열 만들기 2/181921.js
function solution(l, r) {
    const answer = [];
    
    for (let i=l; i<=r; i++) {
        if (i.toString().replaceAll('5', '').replaceAll('0', '') === '') {
            answer.push(i);
        }
    }
    
    if (answer.isEmpty()) {
        answer.push(-1);
    }
    
    return answer;
}
